cmd/resim/commands/project.go

```diff
@@ -48,7 +48,7 @@ var (
 )
 
 const (
-	projectIDKey          = "project-id"
+	projectIDKey          = "identifier"
```

Comment:
i'm not 100% about this as the flag name, but can't think of anything better other than keeping it `--project` as per the others. `project delete --project my_pro` feels okay to me

Reply:
Oh yes. That's a much better idea. THanks!